### PR TITLE
fix: adjust the debug screen according to design specs

### PR DESCRIPTION
**Pull request by trOnk12**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
image

image

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4506311)(4506311)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4506311)(4506311)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4506539)(4506539)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4506539)(4506539)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4523387)(4523387)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4523387)(4523387)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4543518)(4543518)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4543518)(4543518)-dev-release.apk)

**Comment by saleniuk** [comment]
Can we unify the size and colour of icons on this screen? That would look way better 😄

**Comment by saleniuk** [comment]
@trOnk12 can you update these icons and resolve conflicts so we can merge and close this one as well?

**Comment by ohassine** [comment]
Could you please add more details about the issue ? it's not clear from the screenshoot

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4713686)(4713686)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4713686)(4713686)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/settings/SettingsScreen.kt [hunk 1]
```diff
@@ -103,29 +109,48 @@ private fun LazyListScope.folderWithElements(
...
+        actions = {
+            trailingIcon?.let {
+                Icon(
+                    painter = painterResource(id = trailingIcon),
+                    contentDescription = "",
+                    tint = MaterialTheme.wireColorScheme.onSecondaryButtonEnabled,
+                    modifier = modifier
+                        .defaultMinSize(80.dp)
+                        .clickable(onIconPressed)
+                        .then(modifier)
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/settings/SettingsScreen.kt]: Now `modifier` is used twice 😄 Depending on the order it should be either:
```
modifier = modifier
    .defaultMinSize(80.dp)
    .clickable(onIconPressed)
```
or 
```
modifier = Modifier
    .defaultMinSize(80.dp)
    .clickable(onIconPressed)
    .then(modifier)
```
In the second one, new `Modifier` is created and the existing `modifier` is used after all modifiers are set.

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4714074)(4714074)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4714074)(4714074)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4714260)(4714260)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4714260)(4714260)-dev-release.apk)

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/8/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4720513)(4720513)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4720513)(4720513)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/9/) **failed**.

**Comment by AndroidBob** [comment]
Build [10](https://10.10.124.134/job/wire-android-reloaded/job/PR-1036/10/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4721621)(4721621)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4721621)(4721621)-dev-release.apk)

**Pull request merged**